src/v/datalake/record_translator.cc

```diff
@@ -188,6 +189,11 @@ structured_data_translator::build_type(std::optional<resolved_type> val_type) {
     if (val_type.has_value()) {
         val_id = std::move(val_type->id);
         auto& struct_type = std::get<iceberg::struct_type>(val_type->type);
+        // TODO: could make this configurable at some point
```

Comment:
Can we not leave TODOs here and instead give discourse as to *why* do this.

Reply:
done